### PR TITLE
Add second ingress route for CCE to bypass OAuth

### PR DESCRIPTION
**Pull request by jblefkove-va**
https://vasdvp.atlassian.net/browse/CCE-138

### CODE REVIEW

File: products/community-care.yaml [hunk 1]
```diff
@@ -30,6 +30,10 @@ spec:
             backend:
               serviceName: cce-kong
               servicePort: 8082
+          - path: /community-care-vaos/(.*)
```
Comment (bschofield-va) [inline products/community-care.yaml]: sure we don't want something more generic, e.g. `/community-care-internal/(.*)`?

Comment (jblefkove-va) [reply products/community-care.yaml]: There may be `community-care-internal`, but for the moment this is only intended for VAOS. So ad-hoc decided on `/services/community-care-vaos/v0` as the new route

File: products/community-care.conf [hunk 2]
```diff
@@ -4,4 +4,5 @@ DU_NAMESPACE=cce
 DU_DECRYPTION_KEY="$DEPLOYMENT_CRYPTO_KEY"
 DU_HEALTH_CHECK_PATH="/community-care/actuator/health"
 DU_LOAD_BALANCER_RULES[1]="/community-care/*"
+DU_LOAD_BALANCER_RULES[1]="/community-care-vaos/*"
```
Comment (mramirez-va) [inline products/community-care.conf]: Use of the same index here won't work. Needs to be `[2]`

**Comment by bschofield-va** [comment]
https://gph.is/20on3pZ

**Pull request merged**